docs/ref/django-admin.txt

```diff
@@ -722,6 +722,16 @@ be warned that using ``--fake`` runs the risk of putting the migration state
 table into a state where manual recovery will be needed to make migrations
 run correctly.
 
+.. versionadded:: 1.8
+
+.. django-admin-option:: --fake-initial
+
+The ``--fake-initial`` option can be used to allow Django to skip an app's
+initial migration if all database tables with the names of all models created
+by all :class:`~django.db.migrations.operations.CreateModel` operations in that
+migration already exit. This option does not check for matching database
+schema and should thus be used with care.
```

Comment:
This warning could be a bit clearer I think, like the CLI message: "This option does not check for matching database schema beyond matching table names, and so is only safe to use if you are confident that your existing schema matches what is recorded in your initial migration."